app/src/main/java/com/rob729/quiethours/Database/ProfileRoomDatabase.kt

```diff
@@ -22,21 +20,10 @@ abstract class ProfileRoomDatabase : RoomDatabase() {
                 return tempInstance
             }
             synchronized(this) {
-                val instance = Room.databaseBuilder(
-                    context.applicationContext,
-                    ProfileRoomDatabase::class.java,
-                    "Profile_Database"
-                ).addMigrations(MIGRATION_1_2)
-                    .build()
+                val instance = Room.databaseBuilder(context.applicationContext, ProfileRoomDatabase::class.java, "Profile_Database").build()
```

Comment:
Is the master code merged into your branch?

Reply:
No, actually when i pushed the changes then it was showing me conflicts in 4 files and in ProfileRoomDatabase.kt it was showing me migration code in conflict then i removed that code from github and resolved the conflicts.

and in Project.xml file.... when i am removing unnecessary code and then on commiting these unnecessary changes are always there i mean ultimately they are not going..... kindly suggest what to do...

Reply:
Logically think once, is it the right solution to remove those lines of code already present in the RoomDatabse file?
Read about how to solve merge conflicts and then fix the merge conflicts.